nanoserver/1809/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM mcr.microsoft.com/windows/nanoserver:1809
+
```

Comment:
Just out of curiosity, is the netapi32.dll trick no longer needed to run Traefik in a nanoserver container?
For reference: https://github.com/StefanScherer/dockerfiles-windows/blob/main/traefik/Dockerfile.1809#L3 :-)
PS: I haven't done much Windows Containers lately, but would like to know what might have improved since my first attempts.

Reply:
Ah! that would explain the error I've seen with the 1.x container image, I'll do some additional tests

Reply:
I confirm that the issue only affects traefik 1.7, but not 2.4, which is a bit odd. I'll modify the template for 1.x to use the workaround

Reply:
Thanks. Good to know that Traefik 2.4 works without that workaround. 
That was a nasty thing discussed in https://github.com/golang/go/issues/21867 when a Golang program used `user.Current()`, mostly used in `glog`.